src/ejabberd_sm.erl

```diff
@@ -128,7 +128,7 @@
                       priority :: priority(),
                       info     :: info()
                      }.
--type info() :: [info_item()].
+-type info() :: #{info_key() => any()}.
 
 %% Session representation as 4-tuple.
 -type ses_tuple() :: {USR :: jid:simple_jid(),
```

Comment:
It looks like we are leaving an unused type behind, could you validate that nothing else is missed? I only checked this one.